docs/API.md

```diff
@@ -50,4 +54,18 @@ For example: Accessing the data for a chat report will return the data if the re
 - Verbs should be unique and indicate the user's action (as perceived by the user). eg. `Request`, `Open`, `Accept`, `Pay`
   - If a unique verb cannot be used, then use only the standard verbs: `Update`, `Add`, `Delete`
 - Names must NOT include names of parameters eg. Bad: `SignInWithPasswordOr2FA` Good: `SignIn`
+- Names must NOT the word Page or View, eg. Bad: `OpenReportPage` Good: `OpenReport`
 - Names must NOT reveal backend implementation details that the user does not know about eg. `AddVBBA` (users don't know what a VBBA is - verified business bank account) Good: `AddBankAccount`
+- For deep links like `StartAppWhileSignedInAndOpenWorkspace`, create separate commands for each specific scenario like `StartAppWhileSignedIn` and `OpenWorkspace`, where both requests will be triggered in parallel.
+
+## FAQ
+
+### How should error messages be persisted to Onyx?
+
+- The API layer Pusher onyxData should set specific error messages.
+- In cases where the API does not set an error, a generic error message can be set in the API.write onyxData `failureData` block.
+- Previous errors should be cleared in the `optimisticData` when a new request is made
+
+### How should we remove local data from the store if the data no longer exist in the server? eg. local policy data for policies the user is no longer a member of.
+
+Use `Onyx.set(key, null)` for each data (eg. policy) not found in the server.
```

Comment:
Just curious, is this equivalent to `Onyx.merge(key, null)`?

Reply:
`Onyx.merge(key, null)` doesn't do anything. It is basically like saying "merge nothing into key", which wouldn't do anything.